cypress/support/helpers/onDemandRadioTv.js

```diff
@@ -32,11 +33,17 @@ export const getEmbedUrl = ({ body, language, isAmp }) => {
   return isAmp ? `${embedUrl}/amp` : embedUrl;
 };
 
-export const isAvailable = pathEq(
+export const isAvailableOld = pathEq(
```

Comment:
Let's rename this back to `isAvailable` (and rename the other one)

Reply:
OH I forgot to remove that haha. I was just using it to compare the behaviour while coding it

Reply:
Fixing now